app/models/product.rb

```diff
@@ -8,4 +8,6 @@ class Product < ApplicationRecord
   validates :title, presence: true
   validates :description, presence: true
   validates :price, presence: true, numericality: true
+
+  scope :product_all, ->{order(created_at: :desc)}
```

Comment:
đặt cái scope đọc vô khó hiểu vậy
```suggestion
  scope :newest, ->{order created_at: :desc}
```